upload.go

```diff
@@ -125,6 +144,118 @@ type s3repo struct {
 	session    *session.Session
 }
 
+func NewB2Repo(opts options) (*b2repo, error) {
+	r := &b2repo{
+		appKey:            opts.B2AppKey,
+		bucket:            opts.B2Bucket,
+		concurrentUploads: opts.B2ConcurrentUploads,
+		ctx:               context.Background(),
+		endpoint:          opts.B2Endpoint,
+		forcePath:         opts.B2ForcePath,
+		keyID:             opts.B2KeyID,
+		region:            opts.B2Region,
```

Comment:
`region` is not used later in the code and the Backblaze Go API does not seem mention it, even if it seem to exist in the endpoint hostname they give on their WebUI

Reply:
Hi, fixed :)